{{ cookiecutter.package_name }}/tox.ini

```diff
@@ -0,0 +1,87 @@
+[tox]
+envlist =
+    py{36,37,38}-test{,-alldeps,-devdeps}
+    py{36,37,38}-test-numpy{116,117,118}
+    py{36,37,38}-test-astropy{30,40,lts}
+    build_docs
+    codestyle
+requires =
+    setuptools >= 30.3.0
+    pip >= 19.3.1
+isolated_build = true
+
+[testenv]
+
+# Pass through the following environemnt variables which may be needed for the CI
+passenv = HOME WINDIR LC_ALL LC_CTYPE CC CI TRAVIS
+
+# Run the tests in a temporary directory to make sure that we don't import
+# astropy from the source tree
```

Comment:
astropy -> package name

Reply:
changed to 'this package' to just keep it generic